source/devices/AM62PX/android/Release_Specific_Release_Notes.rst

```diff
@@ -119,3 +119,8 @@ Known Issues
      - Some commands don't work from native console, but work via adb shell
      - N/A
      - N/A
+
+   * - `e2e 1410786 <https://e2e.ti.com/support/processors-group/processors/f/processors-forum/1410786/sk-am62p-lp-analog-input-microphone-connected-to-3-5mm-jack-not-working>`_
+     - Analog Input Microphone connected to 3.5mm jack not working
+     - Apply :download:`this patch <0001-audio-update-default-value-for-audio-capture-in.patch>` in :file:`external/tinyhal`, `patch 1 <https://git.ti.com/cgit/android/device-ti-am62x/commit/?h=d-android14-release&id=15025d9dc1039469dd1260c837684edb17f991c1>`_ and `patch 2 <https://git.ti.com/cgit/android/device-ti-am62x/commit/?h=d-android14-release&id=61f86f096f71825df6665bb4e15110a5f5d74aea>`_ in :file:`device/ti/am62x`
```

Comment:
This patch is given in the e2e ticket already mentioned. Not sure it's worth adding to this repo as well. If it is accepted or in a public tree already I wouldn't mind you linking it here.

It's more of a personal opinion, but I don't like carrying temporary patches here. I'll let other maintainers comment though.

Reply:
The patch in question is only merged in https://gitlab.baylibre.com/baylibre/ti/android/aosp/external/tinyhal/-/commit/110fbd20bfd39fec96fcfecc87be3c497a2cb5c3

@vishalmti asked me to attach the patch since (afaik) it was not hosted on git.ti.com.

I can link to https://gitlab.baylibre.com/baylibre/ti/android/aosp/external/tinyhal/-/commit/110fbd20bfd39fec96fcfecc87be3c497a2cb5c3 if that is preferred, please let me know!

Reply:
I talked to @vishalmti on the internal chat and he agreed to add a link to our BayLibre gitlab commit. I will update this.

Reply:
Fixed in v2.